frontend/src/components/FormElements/MyIconButton.jsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import {Button, useTheme} from "@mui/material";
+
+export default function MyIconButton(props) {
+  const { onClick, disabled } = props;
+  const theme = useTheme();
+  return (
+    <Button
+      variant="contained"
+      onClick={onClick}
+      disabled={disabled}
+      color={theme.secondary}
```

Comment:
I noticed this MyIconButton is using color={theme.secondary} wether MyButton is using color="primary". 
Do we want to use primary or secondary for buttons?

It would be good to use theme with MyButton too in case the color palette change in the future.

Reply:
I just linked the styling to the theme on both components.